### ISSUE TITLE
Feature: Synchronize Devices between IoT Hub and Portal database

### ISSUE THREAD

**Issue by kbeaugrand** [CGI-FR/IoT-Hub-Portal#1030]
Blocked by #1029 

We should create some copy items of devices in our database to support much advanced sear, sort queries...

**Comment by kbeaugrand** [comment]
@hocinehacherouf , 

I see that this PR won't sync Edge devices nor LoRaWAN concentrator. Do we need to consider creating a dedicated job to sync them afterward ? 
In that case, Let's create a new issue for them.

**Comment by hocinehacherouf** [comment]
@kbeaugrand:
- Sync edge devices; #1281 
- Sync concentrators: #1287
